commit 34bb7c228110682927727d683df49ccfbfca621c (refs/changes/26/4726/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-09-07T01:19:43+00:00 (1 year, 1 month ago)
    
    TRITON-708 CNS redis pool cap too high
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/server.js b/server.js
index 40d4c07..84be232 100644
--- a/server.js
+++ b/server.js
@@ -40,10 +40,10 @@ var redisPool = new cueball.ConnectionPool({
 	defaultPort: 6379,
 	log: log,
 	spares: 4,
-	maximum: 100,
+	maximum: 12,
 	recovery: {
 		default: {
-			timeout: 1000,
+			timeout: 2000,
 			delay: 500,
 			retries: 5
 		}
